app/assets/javascripts/Components/Result/file_viewer.jsx

```diff
@@ -14,10 +14,21 @@ export class FileViewer extends React.Component {
 
   constructor(props) {
     super(props);
+
+    this.max_sizes = {
+      any: 15e6,
+      image: 15e6,
+      pdf: 15e5,
+      text: 5e6,
+      binary: 5e6,
+      "jupyter-notebook": 15e6,
+    };
```

Comment:
Should probably be moved, but I do not know where to put it.

Reply:
Yeah, it's a big awkward. For now you can leave it in this file, but please pull out into a top-level constant.

In terms of the values, please make the image/PDF/notebook limits up to 50MB, and the text limit up to 100 kB.